ui/cryptomaterial/dropdown.go

```diff
@@ -134,69 +152,78 @@ func (d *DropDown) Changed() bool {
 	return false
 }
 
-func (d *DropDown) layoutActiveIcon(gtx layout.Context, index int) D {
+func (d *DropDown) layoutActiveIcon(gtx C, index int) D {
 	var icon *Icon
 	if !d.isOpen {
 		icon = NewIcon(d.dropdownIcon)
 	} else if index == d.selectedIndex {
 		icon = NewIcon(d.navigationIcon)
 	}
 
-	return layout.E.Layout(gtx, func(gtx C) D {
-		if icon != nil {
-			icon.Color = d.theme.Color.Gray1
-			return icon.Layout(gtx, values.MarginPadding20)
-		}
-		return layout.Dimensions{}
-	})
+	if icon == nil {
+		return D{} // return early
+	}
+
+	if !d.isOpen || d.SelectedItemIconColor == nil {
+		icon.Color = d.theme.Color.Gray1
+	} else {
+		icon.Color = *d.SelectedItemIconColor
+	}
```

Comment:
I find it easier to read the inverted if statement. If you find it simpler too, switch it up:
`if d.isOpen && d.SelectedItemIconColor != nil { // use SelectedItemIconColor }`